modules/ROOT/pages/get-started.adoc

```diff
@@ -4,12 +4,37 @@ After you become familiar with the UI and concepts covered in the xref:index.ado
 
 == Before You Begin
 
-* To access the API Governance console, you must have the *API Governance* > *Governance Administrator* permission.
+Before you begin using API Governance:
 
-* If you have an opt-in banner in the API Governance console, opt in to get additional functionality. See the xref:release-notes::api-governance/api-governance-release-notes.adoc[] for details.
+* <<verify-permissions>>
```

Comment:
Consider providing link text to make these sentences with punctuation (instructions). Or change the intro to indicate that these are sections.

Reply:
Ack

Reply:
Can you explain this standard to me? Are we not supposed to use anchor links as substitutes for instructions, even if the text in them works as instructions? Are we not supposed to make fake mini-TOCs, even for long sections where they might not notice the second part? I don't like make "self-conscious" doc where we mention this section, this documentation, etc., and I'm not quite sure what problem I'm solving.

Reply:
Duke and I went around and around on this with ACB. Yes, you can leave them with the default titles and it's understandable, but it's not a sentence/imperative because it has Title Capitalization and no punctuation. 

One way to get around it is to say something like "Perform the steps in the following sections:" or somesuch. 

It's not a standard and it was just something to "consider".